apps/src/templates/manageStudents/manageStudentsRedux.js

```diff
@@ -1,6 +1,63 @@
 import _ from 'lodash';
 import {SectionLoginType} from '@cdo/apps/util/sharedConstants';
 
+// Response from server after adding a new student to the section.
+export const ADD_STATUS = {
+  "success": "success",
+  "fail": "fail",
+};
+
+// Types of rows in studentData/editingData
+// newStudent looks like a studentRow with isEditing true, but
+// is updated like an add row, since the student has yet to be added.
+export const ROW_TYPE = {
```

Comment:
Another nit: We're not nearly as consistent about this as we should be, but this doesn't follow our naming conventions for enums/constants.

Our [primary styleguide calls for](https://google.github.io/styleguide/javascriptguide.xml?showone=Naming#Naming) `EnumNamesLikeThis`, `CONSTANT_VALUES_LIKE_THIS`.  That would change the `ROW_TYPE` enum to:
```js
export const RowType = {
  ADD: 'addRow',
  NEW_STUDENT: 'newStudentRow',
  STUDENT: 'studentRow',
};
// Used as `RowType.NEW_STUDENT`
```

Same with the `ADD_STATUS` enum, above.